app/views/homepage/index.html.erb

```diff
@@ -52,7 +52,7 @@
 
     <section class="departments-and-policy" aria-labelledby="departments-and-policy-label">
       <div class="govuk-grid-row home-numbers-and-info">
-        <h2 id="departments-and-policy-label" class="govuk-visually-hidden"><%= t('homepage.index.departments_and_policy') %></h2>
+        <h2 id="departments-and-policy-label" class="govuk-visually-hidden"><%= raw(t('homepage.index.departments_and_policy')) %></h2>
```

Comment:
Instead of calling `raw`, could we use the `*_html` key suffix instead?

Reply:
Yes, that's a good point.  We should probably standardise our use of this sometime too.